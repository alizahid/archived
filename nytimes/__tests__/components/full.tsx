import React from 'react'
import { create } from 'react-test-renderer'

import Full from '../../src/components/full'

import { article } from '../fixtures'

it('renders', () => {
  const tree = create(<Full article={article} />)

  expect(tree).toMatchSnapshot()
})
