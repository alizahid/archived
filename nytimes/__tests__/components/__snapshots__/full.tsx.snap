// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<View>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "elevation": 5,
        "shadowColor": "#8E8E93",
        "shadowOffset": Object {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 0.25,
        "shadowRadius": 5,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "https://via.placeholder.com/800x600",
        }
      }
      style={
        Object {
          "backgroundColor": "#000000",
          "height": 562.5,
          "width": 750,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "padding": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#000000",
          "fontFamily": "System",
          "fontSize": 34,
          "fontWeight": "700",
          "letterSpacing": 0.41,
          "lineHeight": 41,
        }
      }
    >
      A test article
    </Text>
    <Text
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#8E8E93",
          "fontFamily": "System",
          "fontSize": 13,
          "fontWeight": "400",
          "letterSpacing": -0.078,
          "lineHeight": 18,
          "marginTop": 10,
        }
      }
    >
      By Ali Zahid
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "marginLeft": -20,
          "marginTop": 10,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginLeft": -20,
              "marginTop": 10,
            },
            Object {
              "marginLeft": 20,
              "marginTop": 0,
            },
          ]
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../src/assets/icons8-calendar.png",
            }
          }
          style={
            Object {
              "height": 20,
              "width": 20,
            }
          }
        />
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#8E8E93",
              "fontFamily": "System",
              "fontSize": 13,
              "fontWeight": "400",
              "letterSpacing": -0.078,
              "lineHeight": 18,
              "marginLeft": 10,
            }
          }
        >
          6/2/2019
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginLeft": -20,
              "marginTop": 10,
            },
            Object {
              "marginLeft": 20,
              "marginTop": 0,
            },
          ]
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../src/assets/icons8-visible.png",
            }
          }
          style={
            Object {
              "height": 20,
              "width": 20,
            }
          }
        />
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#8E8E93",
              "fontFamily": "System",
              "fontSize": 13,
              "fontWeight": "400",
              "letterSpacing": -0.078,
              "lineHeight": 18,
              "marginLeft": 10,
            }
          }
        >
          10
        </Text>
      </View>
    </View>
    <Text
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#000000",
          "fontFamily": "System",
          "fontSize": 17,
          "fontWeight": "400",
          "letterSpacing": -0.408,
          "lineHeight": 22,
          "marginTop": 20,
        }
      }
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris ut laoreet ex. Praesent eu volutpat nibh, vel suscipit eros.
    </Text>
  </View>
</View>
`;
